server/index.ts

```diff
@@ -176,16 +219,32 @@ export const configSchema = schema.object({
       // the login config here is the same as old config `_security.basicauth.login`
       // Since we are now rendering login page to browser app, so move these config to browser side.
       login: schema.object({
-        title: schema.string({ defaultValue: 'Please login to OpenSearch Dashboards' }),
+        title: schema.string({ defaultValue: 'Log in to OpenSearch Dashboards' }),
         subtitle: schema.string({
           defaultValue:
-            'If you have forgotten your username or password, please ask your system administrator',
+            'If you have forgotten your username or password, contact your system administrator.',
         }),
         showbrandimage: schema.boolean({ defaultValue: true }),
         brandimage: schema.string({ defaultValue: '' }),
         buttonstyle: schema.string({ defaultValue: '' }),
       }),
     }),
+    openid: schema.object({
+      login: schema.object({
+        buttonname: schema.string({ defaultValue: 'Log in with single sign-on' }),
+        showbrandimage: schema.boolean({ defaultValue: false }),
```

Comment:
Why are all the `showbrandimage` false by default? Should they be consistent with [this](https://github.com/opensearch-project/security-dashboards-plugin/blob/main/server/index.ts#L97)?

Reply:
Should they be consistent? Let's say, when user does not set any of these settings, for basic auth, the big logo does not show, but the log still shows on the buttons?

Reply:
By default for both basic login button and external login button, they are consistent and they do not show up the logo.